foundations/introduction/introduction_to_web_development.md

```diff
@@ -99,4 +99,4 @@ This section contains helpful links to related content. It isn’t required, so
 *   [FreeCodeCamp: Things I Wish Someone Had Told Me When I Was Learning How To Code](https://www.freecodecamp.org/news/things-i-wish-someone-had-told-me-when-i-was-learning-how-to-code-565fc9dcb329/)
 *   [TechCrunch: Don't Believe Anyone who Tells you Learning to Code is Easy](http://techcrunch.com/2014/05/24/dont-believe-anyone-who-tells-you-learning-to-code-is-easy/)
 *   [Code Quizzes: Deliberate Programming Practice](https://codequizzes.wordpress.com/2013/04/28/deliberate-programming-practice/)
-*   [Roadmap to becoming a web developer in 2022](https://github.com/kamranahmedse/developer-roadmap)
+*   [Roadmap to becoming a web developer in 2023](https://github.com/kamranahmedse/developer-roadmap)
```

Comment:
Thanks for contributing @Trailingslashes. I just had a snoop through the repo. They don't mention a year anywhere so I don't think we should anymore either. That'll help us avoid needing to update this every year.

```suggestion
*   [Roadmap to becoming a web developer](https://github.com/kamranahmedse/developer-roadmap)
```

Reply:
@KevinMulhern, That makes more sense.